build.sbt

```diff
@@ -1,8 +1,8 @@
 lazy val commonSettings = Seq(
   organization := "com.whisk",
-  version := "0.9.8",
-  scalaVersion := "2.12.3",
-  crossScalaVersions := Seq("2.12.3", "2.11.11", "2.10.6"),
+  version := "0.9.9",
+  scalaVersion := "2.13.0",
+  crossScalaVersions := Seq("2.13.0", "2.12.3", "2.11.11"),
```

Comment:
2.12.8 and 2.11.12 are out now too.
https://www.scala-lang.org/download/#other-releases